### PR TITLE
Remove tools directory content

### PR DESCRIPTION
**Pull request by jujaga**
# Description

This PR updates the devops tooling to use the extracted tools repository now found [here](https://github.com/bcgov/nr-showcase-devops-tools).
This PR also contains:
* Backend documentation with database management
* Quick explanation on code structure

This change is required in order to reduce code duplication for devops tooling.

[[SHOWCASE-230]](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-230)

## Types of changes

Bug fix (non-breaking change which fixes an issue)

Documentation (non-breaking change with enhancements to documentation)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-k8vopl-tools.pathfinder.gov.bc.ca/dashboard?id=nr-get-token-pr-80

File: backend/README.md [hunk 1]
```diff
@@ -6,12 +6,19 @@ Natural Resources Get Token (GETOK) will automate the process for Development te
...
 
-Password Management & Authentication
+Authentication & Password Management
 
 * `cryptico-js` - Asymmetric key encryption
 * `generate-password` - Password generator
+* `jsonwebtoken` - JWT parsing library
 * `passportjs` - JWT and OIDC strategies
 
+Database Mangement
```
Comment (loneil) [inline backend/README.md]: typo

File: backend/README.md [hunk 2]
```diff
@@ -85,18 +114,39 @@ Note: for publicKey, you must enter the PEM encoded value with newlines encoded
...
+
+``` sh
+npm run migrate
+```
+
+Once this is done, you should be able to start up the backend application.
+
 ## Commands
 
 After adressing the prerequisites, the following are common commands that are used for this application.
```
Comment (loneil) [inline backend/README.md]: *addressing

File: backend/README.md (hunk 1 above)
Comment (jujaga) [reply backend/README.md]: Fixed

File: backend/README.md (hunk 2 above)
Comment (jujaga) [reply backend/README.md]: Fixed

**Pull request merged**